src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/handle-prototype.js

```diff
@@ -0,0 +1,28 @@
+
+function handlePrototype() {
+    buildFormTree(document.forms.config);
+    // TODO JSON.stringify fails in some circumstances: https://gist.github.com/jglick/70ec4b15c1f628fdf2e9 due to Array.prototype.toJSON
+    var json = Object.toJSON(JSON.parse(document.forms.config.elements.json.value).prototype);
+    if (!json) {
+        return; // just a separator
+    }
+    const url = '/jenkins/pipeline-syntax/generateSnippet' // ${rootURL}/${it.GENERATE_URL}
```

Comment:
This will not work if the context path is set to anything other than `/jenkins`.

Reply:
I do not often work in JS, but there are various ways; IIRC the preferred technique is to define `data` attributes on HTML elements and look those up from JS. Search core sources for examples.